docs/DeveloperGuide.md

```diff
@@ -622,38 +630,192 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file
+   2. Double-click the jar file
        Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
+       
+#### Contact Commands       
```

Comment:
Sorry what are this part for? I thought manual testing is like a quick test of whether the app is working well? 

Also, the Table of Content is not updated if we are going to include this part.